lib/SILOptimizer/Mandatory/AccessMarkerElimination.cpp

```diff
@@ -181,8 +181,13 @@ struct AccessMarkerEliminationPass : SILModuleTransform {
 
       // Markers from all current SIL functions are stripped. Register a
       // callback to strip an subsequently loaded functions on-the-fly.
-      if (!EnableOptimizedAccessMarkers)
-        M.registerDeserializationCallback(prepareSILFunctionForOptimization);
+      if (!EnableOptimizedAccessMarkers) {
+        using NotificationHandlerTy =
+            FunctionBodyDeserializationNotificationHandler;
```

Comment:
I don't see the purpose of the `using` declaration here.

Reply:
80 columns.